### ISSUE TITLE
Test: detached fixture finalizer makes Python 3.9 cleanup a no-op

### ISSUE THREAD

**Issue by Apdelrahman1911** [p2pKit/P2pKit#421]
## Confirmed defect

`ExecutorFixtureTests.setUp()` intentionally detaches the `TemporaryDirectory` finalizer so an ownership or evidence-retention failure preserves the fixture. The later, guarded disposal calls `self.temporary.cleanup()`.

On the native Python 3.9.6 provided with Xcode 26.5, `TemporaryDirectory.cleanup()` only invokes its remover when `_finalizer.detach()` returns truthy. Because setup already detached it, the explicit cleanup is a no-op. The subsequent real-absence assertion fails and leaves all 39 generated fixture roots behind even when their ownership/evidence gates succeeded.

- Baseline: `3bc76f956f8f47447b51a62474fc878b9c43173c`.
- Both relevant lines originated in `731f98db90668f098c6025d3a0df6e863a951197`.
- [Finalizer detachment](https://github.com/p2pKit/P2pKit/blob/3bc76f956f8f47447b51a62474fc878b9c43173c/scripts/tests/run-audit-command-test.py#L1344-L1349).
- [Evidence/ownership-gated disposal](https://github.com/p2pKit/P2pKit/blob/3bc76f956f8f47447b51a62474fc878b9c43173c/scripts/tests/run-audit-command-test.py#L1484-L1490).

Severity: Low; native acceptance-fixture compatibility/resource retirement, not a product protocol or cryptographic defect.

## Actual reproduction and scope

Ordinary-user native arm64 Mac, macOS 26.2, Xcode 26.5 (17F42), native Xcode Python 3.9.6. The documented prerequisite is native Python 3, not a newer-minor-only requirement.

After a successful same-owned-home wrapper bootstrap, the maintained immutable executor ran:

```sh
python3 scripts/tests/run-audit-command-test.py \
  --expected-host macos-arm64 --evidence-dir "$OWNED_EVIDENCE"
```

Here `python3` selects the native Xcode interpreter. The unchanged suite executed **91 tests, 39 teardown failures**. Every latest fixture teardown reported only `Generated fixture directory still exists`. Outer product/stop/final codes were **1/0/1**, receipt validation succeeded, owned survivors and receipt errors were empty, and tracked source was unchanged. This is one cause, not 39 distinct defects.

All 39 final fixture guards and all 7 separately owned sentinels retired. All 4 dependent archives completed; 408 retained entries (793,874 bytes) rehashed against originals and retained copies without mismatch. Earlier intentional failure/recovery records remain separate. Private raw evidence and the failed directories are retained; no deletion or rewriting of failed receipts has occurred.

Newer installed Python implementations use `detach() or exists(name)` and do not exhibit this no-op. The separate Homebrew Python/Expat linkage failure and earlier cold-wrapper stop timeout are historical failures, not caused or repaired by this issue. No simulator, phone, Windows or Intel qualification is claimed.

## Required correction and acceptance

- Keep the detached finalizer, all ownership/archive gates, read-only-file handling, final absence assertion, and failure journals.
- Explicitly perform the standard library remover after gates pass, without reattaching automatic deletion or broadening production cleanup roots.
- Reuse the actual failing native 3.9 run as the negative control; rerun all 91 unchanged native controls on the corrected clean source using that interpreter.
- Require all 91 passes (no skips), valid outer 0/0/0 receipt, unchanged source, no owned survivors, all 39 final fixture removals, intact authentic archives and consumer recovery proofs.
- Independently review final source and execution evidence.
- Reconcile the previously retained exact-path fixtures only after preserving evidence and independently validating ownership; retain the original failure records as history.

Cause-based duplicate review covered the full refreshed issue/PR corpus plus exact searches and relevant histories (#371, #373, #378, #392, #394, #398, #401 and #141); those address different causes and should not be reopened.

**Comment by Apdelrahman1911** [comment]
### Portable continuation checkpoint — 15 September 2026

[Issue-specific preserved work and remaining acceptance](https://github.com/p2pKit/P2pKit/blob/438fbb0dcdcfa2d9aea2167c6df1de8b4969250a/docs/maintenance/nonphysical-continuation-2026-09-15.md#issue-421).

- **Preserved:** `974d51a0ce638b397e6779a0ddaf5b4da8a8bffd`; real native Python3.9 controls91/91, 39 new fixture removals and 39 original-root recoveries independently approved.
- **Remaining:** Preserve #431's later compatible fixture-parent change; final input/check/formal-review/merge reconciliation. Original 39 teardown failures remain recorded.

Use `work/nonphysical-integration-20260915-022112`, not an older topic branch. All 15 campaign tips are consolidated there, **not merged into main**. [Exact source, checks, review scopes/hashes and safe resume instructions](https://github.com/p2pKit/P2pKit/blob/438fbb0dcdcfa2d9aea2167c6df1de8b4969250a/docs/maintenance/nonphysical-continuation-2026-09-15.md).

**Still OPEN.** This documentation upload adds no runtime acceptance, formal PR approval, merge or closure. Required checks and same-issue criteria remain; held execution/downloads and physical-phone work are not restarted. Raw private evidence stays outside GitHub.

Independent review of this documentation/map only: SHA256 `864a8136ea5c24172f4e4fea671478dc2af8e20fa99a5def793b468ebac27736` (not a product/full-gate approval).
